fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LucenOnlineIndexingTest.java

```diff
@@ -717,6 +717,7 @@ protected TerribleMergingIndexMaintainer(final IndexMaintainerState state) {
         @Override
         public CompletableFuture<Void> mergeIndex() {
             final IndexDeferredMaintenanceControl mergeControl = state.store.getIndexDeferredMaintenanceControl();
+            mergeControl.setLastStep(IndexDeferredMaintenanceControl.LastStep.MERGE);
```

Comment:
Note: at some point, outside the scope of this PR, we should rename this class to have the second `e` in `Lucene`

Reply:
Oops. 
Changed it now ✅